src/blog/_posts/2023-08-01-introduction-to-cadence-signals.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Introduction to Cadence Signals
+
+date: 2023-08-06
+author: Chris Qin
+authorlink: https://www.linkedin.com/in/chrisqin0610/
+---
+
+In this blog, we will have a brief overview of Cadence signals, a critical feature that provides a way to directly send data to a running workflow. 
+
+In previous blogs, we have gone through on how to implement a simple HelloWorld workflow, which takes a single string as workflow input and executes another activity to obtain the output. In that, you’ve learned two options to pass data to a workflow
+
+* Via start parameters
+* From return values of activities
+
+There are some limitations on these two options. For start parameters, it can only be passed before workflow execution begins. After the execution is kicked off, there is no way for us to change the input parameters. For activity return values, we can not directly pass value to them. They may come from another third party application at runtime. 
```

Comment:
There are some limitations to these two options.  Start parameters can only be passed ...